input/docs/declarative.md

```diff
@@ -0,0 +1,448 @@
+
+
+## Step 2: Create a Skupper site in each namespace
+
+
+### Deploy the site controller
+
+<div class="code-label session-2">West</div>
+
+    $ oc apply -f https://raw.githubusercontent.com/skupperproject/skupper/0.3/cmd/site-controller/deploy-watch-current-ns.yaml
+
+<div class="code-label session-1">East</div>
+
+    $ oc apply -f https://raw.githubusercontent.com/skupperproject/skupper/0.3/cmd/site-controller/deploy-watch-current-ns.yaml
+
+**NOTE:** To configure Skupper for all namespaces in a cluster:
+
+    $ oc apply -f https://raw.githubusercontent.com/skupperproject/skupper/master/cmd/site-controller/deploy-watch-all-ns.yaml
+
+### Create a Skupper site in the east namespace
+
+<div class="code-label session-1">East</div>
+
+Create a file named `east-site.yml`
+
+    apiVersion: v1
+    data:
+      cluster-local: "false"
+      console: "true"
+      console-authentication: internal
+      console-password: "barney"
+      console-user: "rubble"
+      edge: "true"
+      name: east-site
+      router-console: "true"
+      service-controller: "true"
+      service-sync: "true"
+    kind: ConfigMap
+    metadata:
+      name: skupper-site
+
+Note that the `data.name` value of `east-site` and 
+setting `data.edge` to `true` in East disables network ingress at the
+Skupper router layer.  In our scenario, East needs to establish one
+outbound connection to West.  It does not need to accept any incoming
+connections.  As a result, no network ingress is required in East.
+
+To apply the ConfigMap:
+
+    $ oc apply -f ~/east-site.yml
+
+For more information about each parameter, see the [Site Controller README](https://github.com/skupperproject/skupper/blob/master/cmd/site-controller/README.md).
+
+After completion, you should see deployments named `skupper-service-controller` and `skupper-router`.
+
+### Create a Skupper site in the west Project
+
+<div class="code-label session-2">West</div>
+
+Create a file named `west-site.yml`
+
+    apiVersion: v1
+    data:
+      cluster-local: "false"
+      console: "true"
+      console-authentication: internal
+      console-password: "barney"
+      console-user: "rubble"
+      edge: "false"
+      name: west-site
+      router-console: "true"
+      service-controller: "true"
+      service-sync: "true"
+    kind: ConfigMap
+    metadata:
+      name: skupper-site
+
+Note that the `data.name` value of `west-site`.
+
+To apply the ConfigMap:
+
+    $ oc apply -f ~/west-site.yml
+
+For more information about each parameter, see the [Site Controller README](https://github.com/skupperproject/skupper/blob/master/cmd/site-controller/README.md).
+
+After completion, you should see deployments named `skupper-service-controller` and `skupper-router`.
+
+### Check the installation
+
+If you completed the [Getting Started using the CLI](./index.html) you can check the status of each namespace, use the `skupper status`
+command.
+
+<div class="code-label session-2">West</div>
+
+    $ skupper status
```

Comment:
'skupper status' now includes summary of exposed services. Example:

Skupper is enabled for namespace "skupper" in interior mode. It is not connected to any other sites. It has no exposed services.